woocommerce-gutenberg-products-block.php

```diff
@@ -18,6 +18,8 @@
 
 defined( 'ABSPATH' ) || exit;
 
+define( 'WOOCOMMERCE_BLOCKS_PHASE', 'experimental' );
+
```

Comment:
Currently, the way we define the flag in PHP is hacky, I couldn’t find a way yet to do this via Composer or something, so in the future when you want to build a stable release you need to manually change this.
I'm thinking we use something like the one that changes version numbers. @woocommerce/blocks

Reply:
What about enabling this based on the existence of a built file? To avoid any manual changes.

Reply:
💯 to Mike's reply here! You could maybe use this webpack plugin: https://www.npmjs.com/package/create-file-webpack and create a php file at the end of the build process that basically defines the `WOOCOMMERCE_BLOCKS_PHASE` constant and then you just load that file in the bootstrap.  Note, you should load this as a part of the `src/Domain/Bootstrap` class because the main plugin file (`woocommerce-gutenberg-products-block`) is not loaded when in WooCommerce core.